File: question 2.js
//question 2
function bubble_sort(A){
    let i=0,t,u;
    while (true){
        if (i===A.length){
            break;
        }
        let j=0;
        while (true){
            if (j===A.length-i-1){
                break;
            }
            if (A[j] > A[j+1]){
                t=A[j];
                u=A[j+1];
                A[j+1]=t;
                A[j]=u;
            }
            j++
        }
        i++
    }
    console.log(A)
}
let B=[1231,2312,123,422,235];
bubble_sort(B);

File: question 2.py
def bubble_sort(A):
    for i in range(len(A)):
        for j in range(0,len(A)-i-1):
            if A[j] > A[j+1]:
                A[j], A[j+1] = A[j+1], A[j]
    return A
B=[12,123,21,13,412,23,1,23,0,12]
print(bubble_sort(B))
